eslint.config.js

```diff
@@ -156,10 +168,26 @@ module.exports = [
         }
     },
 
+    // JSON files
+    {
+        name: "eslint/json",
+        files: ["**/*.json"],
+        ignores: ["**/package-lock.json"],
+        language: "json/json",
+        ...json.configs.recommended
+    },
+
+    // JSONC files
+    {
+        name: "eslint/json",
+        files: [".vscode/*.json"],
```

Comment:
```suggestion
        files: [".vscode/*.json","knip.jsonc"],
```
Should we also include the `knip.jsonc` file?